### PR TITLE
added 3 more stylers and tests

### PR DESCRIPTION
**Pull request by GantMan**
only 2 of those items can be tested.   So I tossed them in kitchen sink.

### CODE REVIEW

File: motion/ruby_motion_query/stylers/ui_button_styler.rb [hunk 1]
```diff
@@ -52,6 +52,26 @@ def background_image_highlighted=(value)
         @view.setBackgroundImage value, forState: UIControlStateHighlighted
       end
 
+      def background_image_selected=(value)
+        @view.setBackgroundImage value, forState: UIControlStateSelected
```
Comment (squidpunch) [inline motion/ruby_motion_query/stylers/ui_button_styler.rb]: you mentioned only 2 can be tested, is this the one that was left out?  can't we test it like the other image tests?
```ruby
view.imageForState(UIControlStateSelected).should == @vc.rmq.image.resource('logo')
```

Comment (GantMan) [inline motion/ruby_motion_query/stylers/ui_button_styler.rb]: I was sleepy, and didn't see that.   Will fix and add soon!  ty!

Comment (GantMan) [inline motion/ruby_motion_query/stylers/ui_button_styler.rb]: @squidpunch I see the issue, I needed `backgroundImageForState` and now the tests work!

**Comment by squidpunch** [comment]
:+1:

**Pull request merged**